app/src/main/java/com/fastaccess/ui/modules/profile/org/repos/OrgReposFragment.java

```diff
@@ -148,6 +150,11 @@ public static OrgReposFragment newInstance(@NonNull String username) {
         //Not supported for org profile
     }
 
+    @Override
+    public String getLogin() {
+        return username;
```

Comment:
you could let go of the username variable and let the getLogin() return `getArguments().getString(BundleConstant.EXTRA);`